scripts/build_nrf_unix.sh

```diff
@@ -7,55 +7,20 @@ PROJECT_ROOT=$LF_BIN_DIRECTORY/..
 echo "starting NRF generation script into $LF_SOURCE_GEN_DIRECTORY"
 echo "pwd is $(pwd)"
 
-# Copy platform into /core
-cp $PROJECT_ROOT/platform/lf_nrf52_support.c $LF_SOURCE_GEN_DIRECTORY/core/platform/
-cp $PROJECT_ROOT/platform/lf_nrf52_support.h $LF_SOURCE_GEN_DIRECTORY/core/platform/
-cp $PROJECT_ROOT/platform/platform.h $LF_SOURCE_GEN_DIRECTORY/core/
-cp $PROJECT_ROOT/platform/reactor.c $LF_SOURCE_GEN_DIRECTORY/core/
-cp $PROJECT_ROOT/platform/reactor_common.c $LF_SOURCE_GEN_DIRECTORY/core/
-
-# Copy platform into /include/core
-# TODO: Why are there two generated core dirs
-cp $PROJECT_ROOT/platform/lf_nrf52_support.c $LF_SOURCE_GEN_DIRECTORY/include/core/platform/
-cp $PROJECT_ROOT/platform/lf_nrf52_support.h $LF_SOURCE_GEN_DIRECTORY/include/core/platform/
-cp $PROJECT_ROOT/platform/platform.h $LF_SOURCE_GEN_DIRECTORY/include/core/
-cp $PROJECT_ROOT/platform/reactor.c $LF_SOURCE_GEN_DIRECTORY/include/core/
-cp $PROJECT_ROOT/platform/reactor_common.c $LF_SOURCE_GEN_DIRECTORY/include/core/
-
 # Copy the lib directory.
 cp -r $PROJECT_ROOT/lib/* $LF_SOURCE_GEN_DIRECTORY/lib
```

Comment:
Is this still necessary?

Reply:
This copies some buckler specific lib dependencies like romi.c to the src-gen so still think its required.